examples/image-carousel/elements/ImageTile.js

```diff
@@ -0,0 +1,23 @@
+const Tram = require('../../../tram-one')
+
+const html = Tram.html()
+
+const imgStyle = `
+  width : 200px;
+  height : 150px;
+  margin : auto;
+  display : block;
+`
+const divStyle = `
+  padding : 20px;
+  display: flex;
+  cursor : pointer;
+`
+
+module.exports = (attrs, children) => {
+  return html`
+    <div style=${divStyle}>
+      <abbr title=${attrs.value.alt}><img style=${imgStyle} src=${attrs.value.image} alt=${attrs.value.alt} onclick=${attrs.onSelect}/></abbr>
```

Comment:
Huh, I never knew of the `<abbr>` tag before... That being said, it's probably not necessary for this example.